pkgs/top-level/all-packages.nix

```diff
@@ -23332,4 +23332,5 @@ in
     stdenv = crossLibcStdenv;
   };
 
+  nsis = callPackage ../development/tools/nsis { };
```

Comment:
You put this in the very end of all-packages. There is an appropriate section:
https://github.com/NixOS/nixpkgs/blob/198b7ea83ad53fbb679768675b23978b0d090c95/pkgs/top-level/all-packages.nix#L8296

Reply:
Wasn't aware of that, thanks!